tests/operators/models/test_models.py

```diff
@@ -0,0 +1,182 @@
+import pytest
+import torch
+
+from mrpro.operators.models import InversionRecovery
+from mrpro.operators.models import Molli
+from mrpro.operators.models import SaturationRecovery
+from tests import RandomGenerator
+
+
+def test_saturation_recovery_t_0():
+    """Test for saturation recovery at t=0.
+
+    Checking that idata output tensor is close to 0.
+    """
+
+    # Random qdata tensor
+    random_generator = RandomGenerator(seed=0)
+    Nz, Ny, Nx = 100, 100, 100
+    num_coils = 4
+    other = 10
+    random_tensor = random_generator.float32_tensor(size=(2, other, num_coils, Nz, Ny, Nx), low=1e-10)
+
+    # Tensor of TI
+    ti = torch.tensor([0.0])
+
+    # Generate signal model and torch tensor for comparison
+    model = SaturationRecovery(ti)
+    image = model.forward(random_tensor)
+    zeros = torch.zeros_like(image)
+
+    # Assert closeness to zero
+    torch.testing.assert_close(image, zeros)
+
+    assert image is not None
```

Comment:
We do not need this and the next assert because if image is None the above asserting will also fail. To ensure the correct size we could create a zeros-image with the correct shape rather than from image and then `torch.testing.assert_close(image, zeros)` will fail if the shape is wrong.